targets/HUDL/main.cpp

```diff
@@ -107,36 +93,20 @@ int main() {
     DEV::Timerf3xx timer(TIM2, 160);
 
     //create the RPDO node
-    IO::GPIO* devices[deviceCount];
+    IO::GPIO* devices[DEVICE_COUNT];
 
     IO::GPIO& regSelect = IO::getGPIO<IO::Pin::PA_3>(EVT::core::IO::GPIO::Direction::OUTPUT);
```

Comment:
```suggestion
    IO::GPIO& regSelect = IO::getGPIO<IO::Pin::PA_3>(IO::GPIO::Direction::OUTPUT);
```
Is the EVT::core necessary on this lines since the IO namespace is defined above.